packages/fxa-auth-server/test/local/payments/fixtures/customer1.json

```diff
@@ -53,6 +54,7 @@
         "current_period_end": 1568573770,
         "cancel_at_period_end": false,
         "ended_at": null,
+        "latest_invoice": "628031D-0002",
```

Comment:
The latest invoice is an invoice id, it will have a pattern similar to `in_1FObmFKb9q6OnNsLTMm1ERZE`

Reply:
There are invoice samples in the fixtures folder.